py_mmd_tools/nc_to_mmd.py

```diff
@@ -620,12 +622,28 @@ def to_mmd(self, *args, **kwargs):
 
         for key in mmd_yaml:
             self.metadata[key] = self.get_acdd_metadata(mmd_yaml[key], ncin, key)
+
+        # Set storage_information
+        md5hasher = FileHash('md5')
+        fchecksum = md5hasher.hash_file(self.netcdf_product)
+        self.metadata['storage_information'] = {
+                'file_name': os.path.basename(self.netcdf_product),
+                'file_location': os.path.abspath(self.netcdf_product).replace(os.path.basename(self.netcdf_product),''),
+                'file_format': 'NetCDF-CF',
+                'file_size': pathlib.Path(self.netcdf_product).stat().st_size / (1024 * 1024),
+                'file_size_unit': 'MB',
+                'checksum': fchecksum,
+                'checksum_type': '%ssum'%md5hasher.hash_algorithm,
```

Comment:
Why are you adding sum to the hash_algorithm type?

Reply:
that is needed as an attribute to the checksum field in MMD

Reply:
Yes I understand that. But here the `md5hasher.hash_algorithm` returns `md5`
And the `checksum_type` becomes `md5sum`. Why do you add `sum` to `md5` for the `checksum_type`?

Reply:
that's required by MDD: "Checksums can be md5sum, shasum (sha256sum, sha384sum, sha512sum) and other higher order approaches."